### PR TITLE
feature 89 sprint view card

### PR DESCRIPTION
**Pull request by yuusufisse**

### CODE REVIEW

File: src/components/home/sprint-view-card.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { Card, CardContent, Typography } from "@mui/material";
+import { Link } from "react-router-dom";
+import strings from "../../localization/strings";
+import UserRoleUtils from "../../utils/user-role-utils";
+
+const SprintViewCard = () => {
```
Comment (Jdallos) [inline src/components/home/sprint-view-card.tsx:6]: Missing TSDoc comments e.g. see balance card component comments

File: src/components/home/sprint-view-card.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    const adminMode = UserRoleUtils.adminMode();
```
Comment (Jdallos) [inline src/components/home/sprint-view-card.tsx:7]:
```suggestion
    const adminMode = UserRoleUtils.adminMode();
    
```

File: src/components/home/sprint-view-card.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import { Card, CardContent, Typography } from "@mui/material";
```
Comment (Jdallos) [inline src/components/home/sprint-view-card.tsx:1]: All indentation in this file seems to be double what it should be (2 spaces)

File: src/components/home/sprint-view-card.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                background: "#efefef"
+              }
+            }}
+          >
+            {adminMode ? (
+              <CardContent>
+              <Typography variant="h6" fontWeight={"bold"} style={{ marginTop: 6, marginBottom: 3 }}>
+              {strings.sprint.sprintview}
+              </Typography>
+              <Typography variant="body1">{strings.placeHolder.notYetImplemented}</Typography>
```
Comment (Jdallos) [inline src/components/home/sprint-view-card.tsx:25]: Fix indentation

File: src/components/home/sprint-view-card.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+              </CardContent>
+            ) : (
+                <CardContent>
+                <Typography variant="h6" fontWeight={"bold"} style={{ marginTop: 6, marginBottom: 3 }}>
+                {strings.sprint.sprintview}
+                </Typography>
+                <Typography variant="body1">{strings.placeHolder.notYetImplemented}</Typography>
+                </CardContent>
```
Comment (Jdallos) [inline src/components/home/sprint-view-card.tsx:33]: Fix indentation

File: src/components/screens/admin-screen.tsx [hunk 6]
```diff
@@ -9,6 +10,9 @@ const AdminScreen = () => (
   <Grid container spacing={2}>
     <Grid item xs={12} sm={6}>
       <BalanceCard />
+      <Grid item xs={12} style={{marginTop: "16px"}}>
+          <SprintViewCard />
+        </Grid>
```
Comment (Jdallos) [inline src/components/screens/admin-screen.tsx:15]: Fix indentation

File: src/components/screens/admin-screen.tsx [hunk 7]
```diff
@@ -9,6 +10,9 @@ const AdminScreen = () => (
   <Grid container spacing={2}>
     <Grid item xs={12} sm={6}>
       <BalanceCard />
+      <Grid item xs={12} style={{marginTop: "16px"}}>
```
Comment (Jdallos) [inline src/components/screens/admin-screen.tsx:13]:
```suggestion
      <Grid item xs={12} style={{ marginTop: "16px" }}>
```

File: src/components/screens/home-screen.tsx [hunk 8]
```diff
@@ -10,6 +11,9 @@ const HomeScreen = () => {
     <Grid container spacing={2}>
       <Grid item xs={12} sm={6}>
         <BalanceCard />
+        <Grid item xs={12} style={{marginTop: "16px"}}>
```
Comment (Jdallos) [inline src/components/screens/home-screen.tsx:14]:
```suggestion
        <Grid item xs={12} style={{ marginTop: "16px" }}>
```

File: src/components/screens/sprint-view-screen.tsx [hunk 9]
```diff
@@ -0,0 +1,11 @@
+
+/**
+ * Sprint View screen component
+ */
+const SprintViewScreen = () => {
+  return (
+    <h1>Sprint View Screen</h1>
```
Comment (Jdallos) [inline src/components/screens/sprint-view-screen.tsx:7]: Localize

File: src/components/screens/sprint-view-screen.tsx [hunk 10]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (Jdallos) [inline src/components/screens/sprint-view-screen.tsx:1]:
```suggestion
```

**Review by Jdallos — CHANGES_REQUESTED** [review]

**Review by Kseniia-Koshkina — APPROVED** [review]

**Review by DZotoff — APPROVED** [review]
Best code ever!

File: src/components/home/sprint-view-card.tsx [hunk 11]
```diff
@@ -19,17 +23,17 @@ const SprintViewCard = () => {
...
             {adminMode ? (
               <CardContent>
-              <Typography variant="h6" fontWeight={"bold"} style={{ marginTop: 6, marginBottom: 3 }}>
-              {strings.sprint.sprintview}
-              </Typography>
-              <Typography variant="body1">{strings.placeHolder.notYetImplemented}</Typography>
-              </CardContent>
-            ) : (
-                <CardContent>
                 <Typography variant="h6" fontWeight={"bold"} style={{ marginTop: 6, marginBottom: 3 }}>
```
Comment (Jdallos) [inline src/components/home/sprint-view-card.tsx]: As a general rule, if a component has 3 or more props they should each be indented on a new line, this improves code readability.

File: src/components/home/sprint-view-card.tsx [hunk 12]
```diff
@@ -19,17 +23,17 @@ const SprintViewCard = () => {
...
                 </Typography>
                 <Typography variant="body1">{strings.placeHolder.notYetImplemented}</Typography>
+              </CardContent>
+            ) : (
+                <CardContent>
+                  <Typography variant="h6" fontWeight={"bold"} style={{ marginTop: 6, marginBottom: 3 }}>
+                    {strings.sprint.sprintview}
+                  </Typography>
+                  <Typography variant="body1">{strings.placeHolder.notYetImplemented}</Typography>
                 </CardContent>
```
Comment (Jdallos) [inline src/components/home/sprint-view-card.tsx]: This should be indented the same amount at the condition above- so indented 2 spaces less.

File: src/components/home/sprint-view-card.tsx [hunk 13]
```diff
@@ -3,8 +3,12 @@ import { Link } from "react-router-dom";
...
 import UserRoleUtils from "../../utils/user-role-utils";
 
+/**
+ * SprintView card component
+ */
 const SprintViewCard = () => {
     const adminMode = UserRoleUtils.adminMode();
+    
     return (
         <Link
```
Comment (Jdallos) [inline src/components/home/sprint-view-card.tsx]: Everything between link to link should be indented 2 spaces less

File: src/components/screens/sprint-view-screen.tsx [hunk 14]
```diff
@@ -1,10 +1,11 @@
+import strings from "../../localization/strings";
 
 /**
  * Sprint View screen component
  */
 const SprintViewScreen = () => {
   return (
-    <h1>Sprint View Screen</h1>
+    <h1>{strings.sprint.sprintviewScreen}</h1>
```
Comment (Jdallos) [inline src/components/screens/sprint-view-screen.tsx:8]:
```suggestion
    <h1>{ strings.sprint.sprintviewScreen }</h1>
```

**Review by Jdallos — APPROVED** [review]
Couple of changes and approved

**Pull request merged**